src/checkbox/index.scss

```diff
@@ -19,21 +20,42 @@
 
     background: $color-gray-08;
 
-    border: $border-dark;
+    border: $border-darkgray;
     border-radius: $border-radius-s;
   }
 
+  &.is-checked .c-checkbox-control {
+    border-color: $color-green-02;
+    background: $color-green-02;
+
+    input[type='checkbox'] {
+      &:checked + .c-checkbox-state {
```

Comment:
`input...` and `&:checked` seem redundant. simply use `&.is-checked .c-checkbox-state` for this